src/pysdmx/io/xml/__structure_aux_reader.py

```diff
@@ -551,8 +551,8 @@ def __format_con_id(self, concept_ref: Dict[str, Any]) -> Dict[str, Any]:
         return rep
 
     @staticmethod
-    def __format_relationship(json_rel: Dict[str, Any]) -> Optional[str]:
-        att_level = None
+    def __format_relationship(json_rel: Dict[str, Any]) -> str:
+        att_level = "D"
 
         for scheme in [DIM, PRIM_MEASURE, MEASURE, DFW, OBSERVATION]:
```

Comment:
These tokens (DIM, DFW, etc.) seem to be imported from pysdmx.io.xml.sdmx21? Is that the case? If so, this is odd in a file that should be version-agnostic, isn't it? I mean, maybe the structure file being read is an SDMX-ML 3.0?